packages/spectral/src/serverTypes/convertIntegration.ts

```diff
@@ -55,6 +55,9 @@ import {
 import merge from "lodash/merge";
 import { createInvokeFlow } from "./perform";
 import { createDebugContext, logDebugResults } from "./context";
+import path from "path";
+import { readFileSync } from "fs";
```

Comment:
I get scolded every time I use one of these synchronous versions even though it likely makes _zero_ difference in reality. So I'm fine with this but be warned 😄